lib/floki.ex

```diff
@@ -104,6 +104,28 @@ defmodule Floki do
 
   defdelegate parse_document(document), to: Floki.HTMLParser
 
+  @doc """
+  Parses a HTML Document from a string.
+
+  Similar to `c:parse_document/1`, but raises `Floki.ParseError` if there was an
```

Comment:
Why using `c:` instead of `Floki.`?

Reply:
I mixed up the prefix for documentation in this case. I went ahead and fixed it, thanks! 😄